oqs-template/oqsprov/oqsprov_keys.c/oqsnames.fragment

```diff
@@ -30,9 +30,9 @@
 static oqs_nid_name_t nid_names[NID_TABLE_LEN] = {
 #ifdef OQS_KEM_ENCODERS
 {% for kem in config['kems'] %}
-  { 0, "{{ kem['name_group'] }}", {{ kem['oqs_alg'] }}, KEY_TYPE_KEM, {{ kem['bit_security'] }} },
+  { 0, "{{ kem['name_group'] }}", {{ kem['oqs_alg'] }}, KEY_TYPE_KEM, {{ kem['bit_security'] }}, 0 },
 {%- for hybrid in kem['hybrids'] %}
-  { 0, "{{hybrid['hybrid_group']}}_{{ kem['name_group'] }}", {{ kem['oqs_alg'] }}, {% if hybrid['hybrid_group'].startswith('p') -%} KEY_TYPE_ECP_HYB_KEM {% else %} KEY_TYPE_ECX_HYB_KEM {% endif %}, {{ kem['bit_security'] }} },
+  { 0, "{% if 'standard_name' in hybrid %}{{hybrid['standard_name']}}{% else %}{{ hybrid['hybrid_group'] }}_{{ kem['name_group'] }}{% endif %}", {{ kem['oqs_alg'] }}, {% if hybrid['hybrid_group'].startswith('p') -%} KEY_TYPE_ECP_HYB_KEM {% else %} KEY_TYPE_ECX_HYB_KEM {% endif %}, {{ kem['bit_security'] }}, {% if 'fips_standard' in kem and not hybrid['hybrid_group'].startswith('p') %}1{% else %}0{% endif %} },
```

Comment:
What's the logic with the `startswith('p')` test?

Reply:
`not hybrid['hybrid_group'].startswith('p')` captures all hybrids with curves that don't start with 'p', i.e. not p256/p384/p521. So it sets the 'reverse' flag to 1 if the pq-kem is a fips_standard and the curve is x25519 or x448.

Reply:
But sometimes p256 is named SecP256. I guess the root of my question is if there's some kind of fragile implicit logic here that might not be obvious to someone else down the line.

Reply:
I reused the logic that was already in the template for matching with startswith('p'), and at least in the template the curves are not named secpXXX. But I agree this is fragile, I can change this here to explicitly match x25529 and x448.

Reply:
Updated in 77214a4

Reply:
> there's some kind of fragile implicit logic here that might not be obvious to someone else down the line

Here and in many other places of the code. Mea culpa.

Getting sth done with little time available always has been priority #1. I tried to be conscientious but I've been pretty much alone on this -- and did many other things in parallel (more and more people do more and more of the things I've been doing "behind the scenes" and still OQS keeps dropping feature/functions I enabled before). Hence I've been arguing/looking for "someone else" (ideally more than 1 :) -- and not down the line, but since years -- and more and more vehemently since PQCA started to market this as "product ready" despite a clear lack of technical understanding (or lack of concern for possible problems "down the line") what this really entails. Sorry, I am a fallible human with limited energy (and you know who/what has drained most of it, @dstebila ...).